inquire/src/prompts/path_select/modes.rs

```diff
@@ -0,0 +1,50 @@
+//! Prompt modes 
+//! 
+//! Everything here should be exposed to the (library) user
+use strum::{Display, EnumCount, FromRepr};
```

Comment:
I think that, for the single purpose of enriching PathSortingMode, it is better to implement these methods ourselves than pulling in the dependency